loan-payoff/REPWRITERSPECS/BANNO.LOAN.PAYOFF.V1.POW

```diff
@@ -833,6 +867,84 @@ PROCEDURE DISPLAYRESULTS
  PRINT "},"
  NEWLINE
  CALL GETCOLLATERAL
+
+ IF PAYOFFFEECOUNT>0 THEN
+  DO
+   PRINT "   "+Q+"payoffFees"+Q+":["
+   NEWLINE
+   FOR TMPLOOP=1 TO PAYOFFFEECOUNT
+    DO
+     TMPCHR=FORMAT("#######9.99",PAYOFFFEEAMT(TMPLOOP))
+     CALL REMOVELEADINGSPACES
+     PRINT "    {"+Q+"feeAmount"+Q+":"+Q+TMPCHR+Q+","
+     PRINT Q+"feeDescription"+Q+":"+Q+PAYOFFFEEDESC(TMPLOOP)+Q+"},"
```

Comment:
I think so.  I was a little confused because in code, this is ok but I think the JSON will not like it.  I can't even save it with a trailing space to mock test it :).  Fixed.